google/cloud/spanner/samples/samples.cc

```diff
@@ -1779,7 +1770,7 @@ void QueryNewColumn(google::cloud::spanner::Client client) {
     std::cout << "AlbumId: " << std::get<1>(*row) << "\t";
     auto marketing_budget = std::get<2>(*row);
     if (marketing_budget) {
-      std::cout << "MarketingBudget: " << marketing_budget.value() << "\n";
+      std::cout << "MarketingBudget: " << *marketing_budget << "\n";
```

Comment:
nit: unrelated?

Reply:
Yes.  And no ... I should have mentioned in the description an attempt to make the samples more `StatusOr`-idiomatic.  The underlying issue is that I probably should not have attempted any cleanup in a feature branch, but it is difficult to resist.